azurerm/internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -81,9 +81,10 @@ func resourceLinuxVirtualMachine() *schema.Resource {
 			"os_disk": virtualMachineOSDiskSchema(),
 
 			"size": {
-				Type:         schema.TypeString,
-				Required:     true,
-				ValidateFunc: validation.StringIsNotEmpty,
+				Type:             schema.TypeString,
+				Required:         true,
+				ValidateFunc:     validation.StringIsNotEmpty,
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
This is hiding an issue where the casing is incorrect in the Terraform Configuration, in the case of `TestAccLinuxVirtualMachine_otherEncryptionAtHostEnabled` the actual sku name is `Standard_DS3_v2` rather than `Standard_DS3_V2` - so we should update this to match

Whilst Azure doesn't differentiate these two here, unfortunately Terraform Core expects the value in the configuration to exactly match the value returned from the API - so rather than doing this (and causing a `Plan didn't match the apply` down the road), can we revert this and fix the casing here:

```suggestion
```